main/src/ca/uwaterloo/flix/language/phase/typer/ConstraintGen.scala

```diff
@@ -444,20 +444,22 @@ object ConstraintGen {
 
       case e: Expr.RestrictableChoose => RestrictableChooseConstraintGen.visitRestrictableChoose(e)
 
-      case KindedAst.Expr.Tag(symUse, exp, tvar, loc) =>
+      case KindedAst.Expr.Tag(symUse, exps, tvar, evar, loc) =>
```

Comment:
We might consider dividing up `ApplyTag` and `Tag`, but that can be future work.

Reply:
This could conceptually be called ApplyTag. There is only fully applied tags just like with defs